Resources/Private/JavaScript/media-module/src/components/Presentation/Column.tsx

```diff
@@ -6,7 +6,7 @@ import { createUseMediaUiStyles, MediaUiTheme } from '@media-ui/core/src';
 const useStyles = createUseMediaUiStyles((theme: MediaUiTheme) => ({
     column: {
         display: 'grid',
-        gridAutoRows: 'min-content',
+        gridAutoRows: 'minmax(0, min-content)',
```

Comment:
What is this solving?

Reply:
Before, a grid row was not able to define its own height but instead grew indefinitely with its content. With this change a row can set its height value which allow scrolling only in this row e.g.